Cloudflare/gateway_dns/ingest/parser.yml

```
@@ -20,10 +20,18 @@ stages:
           device.id: '{{json_event.message.DeviceID}}'
           dns.question.name: '{{json_event.message.QueryName}}'
           dns.question.type: '{{json_event.message.QueryTypeName}}'
+      - set:
+          dns.resolved_ip: '{{json_event.message.ResolvedIPs}}'
+        filter: '{{json_event.message.ResolvedIPs != []}}'
```

```suggestion
      - set:
          dns.resolved_ip: >
            [
            {% for ip in json_event.message.ResolvedIPs %}
              {% if ip | is_ipaddress %}
                '{{ip}}',
              {% endif %}
            {% endfor %}
            ]
        filter: '{{json_event.message.ResolvedIPs != []}}'
```
The is_ipaddress filter is needed on every ip adress in order to check if they are valid.